test/performance/odp_timer_perf_run.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+#
+# Copyright (c) 2020, Nokia
+# All rights reserved.
+#
+# SPDX-License-Identifier:	BSD-3-Clause
+#
+
+TEST_DIR="${TEST_DIR:-$(dirname $0)}"
+
+echo odp_timer_perf: odp_schedule overhead mode
+echo ===============================================
+
+$TEST_DIR/odp_timer_perf${EXEEXT} -m 0 -c 1
+if [ $? -ne 0 ]; then
+	echo odp_timer_perf -m 0: FAILED
+	exit $?
```

Comment:
`$?`is the exit status of the last executed command, which is in this case the echo line.

Reply:
fixed in v4. Also fixed the same in odp_scheduling_run.sh.